### ISSUE TITLE
Captions button does not announce its ‘selected’ status

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-media#250]
**Current behaviour:** 
The Closed Captions button toggles two states, active and inactive. The difference between these states is shown visually when the button changes its appearance, but this is not set programmatically and so cannot be understood by screen readers or similar assistive tech.

active > ![cc_active](https://user-images.githubusercontent.com/7045330/215159806-6ca31952-d0f8-49b6-bbc0-00d5f63611b6.png)           inactive > ![cc_inactive](https://user-images.githubusercontent.com/7045330/215159807-127ef9a3-d483-4c81-9342-6086f29229d5.png)

A blind user may wish to have closed captions active as these can be read by a screen reader and will include any additional on-screen text that is not announced audibly during the video. This is also of benefit for deaf-blind users who may be using a braille reader and cannot see or hear anything on the video itself.

**To replicate:** 
With a screen reader active, activate the Closed Captions control and listen to the outputs. 

**Expected behaviour:** 
The control should have an aria attribute such as aria-pressed, allowing it to announce
its state when active, for example “Closed captions, pressed”.

**Recommended Fix:** 
Add an aria-pressed attribute to this control which changes state (from “false” to “true”) when activated.
